services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/volumes.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from fastapi import APIRouter, Depends
+from fastapi import Path as PathParam
+from fastapi import status
+from models_library.volumes import VolumeCategory
+from pydantic import BaseModel
+
+from ..modules.mounted_fs import MountedVolumes
+from ..modules.volume_files import set_volume_state
+from ._dependencies import get_mounted_volumes
+
+router = APIRouter()
+
+
+class PatchVolumeItem(BaseModel):
+    requires_saving: bool
+    was_saved: Optional[bool]
+
+
+@router.patch(
+    "/volumes/{id}",
+    summary="Updates the state of the volume",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def patch_volume_state(
```

Comment:
it is not a patch since you actually replace it right?
then it is a PUT call or else you need to change your PATCH pattern to allow for selectively changing stuff.